projects/packages/changelogger/src/Utils.php

```diff
@@ -143,11 +146,49 @@ public static function loadChangeFile( $filename, &$diagnostics = null ) {
 			throw $ex;
 		}
 		$diagnostics['lines'][''] = $line + strspn( $contents, "\n" );
-		$ret['']                  = trim( $contents );
 
+		$contents = trim( $contents );
+		if ( ! empty( $input_options['add-pr-num'] ) && $contents && $output && $formatter ) {
```

Comment:
Why did you decide to put it here instead of doing it in `loadAllChanges()` like the call to `getTimestamp()`?